index.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+	$('#myForm').on('submit', function(e) {
+		e.preventDefault();
+		$("#output").empty();
+		var searchTerm = $("#searchTerm").val();
+		var url = 
+		"https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser=" 
+		+ searchTerm + "&ucdir=older&uclimit=5&callback=?"; 
+		$.ajax({
+			url: url,
+			type: 'GET',
+			contentType: "application/json; charset=utf-8",
+			async: false,
+			dataType: "jsonp",
+			headers: { 'Api-User-Agent': 'useredits-tool/1.0' },
+			success: function(data, status, jqXHR) {
+				if(data.error != null) {
+					alert(data.error.info);
+				}
+				else {
+					arr = data.query.usercontribs;
+					var str = "";
+					for(var i=0;i<arr.length;i++) {
+						str +=
+						"<div class='panel panel-default col-xs-4' > <div class='panel-body'>Title: " 
+						+ arr[i].title + " (<a href='https://en.wikipedia.org/w/index.php?" + 
+						arr[i].title.replace(/ /g,"_") + "&oldid=" + arr[i].revid + "&diff=prev'>diff</a>)"
+						 + "<br>Timestamp: " + arr[i].timestamp + "<br>Comment: " + arr[i].comment + "</div></div>";
```

Comment:
Likewise, it is not safe to insert a comment into HTML, it can contain a `<script>` tag that loads malicious javascript for example.

Reply:
More realistically, comments with HTML in them will mess up the display. Try searching for the contributions of user `S`.

Reply:
Removed comment section.

Reply:
It's very useful for navigating edits, though. It should be escaped, not removed.

Reply:
Done.